package.json

```diff
@@ -1,13 +1,42 @@
 {
   "name": "dart.dev",
   "version": "0.0.0",
-  "private": false,
-  "description": "Source code for https://dart.dev",
+  "private": true,
```

Comment:
do we want to publish?

Reply:
Nope since the site isn't a package. Setting "private" to `true` ensures no one will accidentally try to do so hhh.